opam/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+name: "hlarp"
+version: "0.0.0"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: ["Isaac Hodes <isaachodes@gmail.com>"
+          "Leonid Rozenberg <leonidr@gmail.com>"]
+homepage: "https://github.com/hammerlab/hlarp"
+bug-reports: "https://github.com/hammerlab/hlarp/issues"
+license: "Apache-2.0"
+build: [
+  [make]
+]
+install: [make "install"]
+remove: [make "uninstall"]
+depends: [
+  "ocamlfind"
+  "cmdliner"
+  "nonstd"
+  "re"
+  "re.posix"
+  "re.glob"
```

Comment:
1. I don't think that `re.posix` and `re.glob` are independent opam packages, `re` should be fine.
2. `ocamlfind` can have a `{build}` qualifier on it.
3. We also depend on `ocamlbuild` which also should have a `{build}` qualifier on it.